### ISSUE TITLE
Google authentication

### ISSUE THREAD

**Issue by softsan** [CrossGeeks/SocialMediaAuthenticationSample#2]
Hi,

I see facebook and Instagram but no google authentication.

so is there any way?
btw, this plugin uses any UIWebview in iOS? if so we have a deprecated warning from apple app store and can't accept the binary later on.

Thanks

**Comment by MobilyteApps** [comment]
yes, i think this plugin uses UIWeview and this is deprecated by apple app store..

**Comment by Jefe21054** [comment]
I have troubles in iOS with Google login, i have a Xamarin Forms project and it opens a Web View and it makes the login with google, but never go inside the app just stay at the login page and never kick the code on the Viewmodel like Android. I'm new developing on Xamarin, so need a bit of help please.

**Comment by KevinAnass** [comment]
any update?
